app/javascript/dashboard/routes/dashboard/helpcenter/pages/categories/NameEmojiInput.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="flex items-center relative">
+    <div
+      v-if="icon"
+      class="absolute cursor-pointer z-20 p-0.5 top-7 ltr:left-[38px] rtl:right-[38px] bg-red-100 hover:bg-red-200/50 dark:bg-red-800/30 dark:hover:bg-red-800/60 text-red-400 dark:text-red-500 hover:text-red-700 dark:hover:text-red-300 rounded-xl"
+      @click="onClickInsertEmoji('')"
+    >
+      <fluent-icon size="12" icon="delete" type="outline" />
+    </div>
```

Comment:
instead of having this wrapped inside the input visually, we can have it separate, something like this

#### Default

![image](https://github.com/chatwoot/chatwoot/assets/18097732/c82a6a8c-3a96-4d7f-9175-85eabf10d580)

#### on hover

![image](https://github.com/chatwoot/chatwoot/assets/18097732/513d2d45-fc67-40bc-80c6-4e8d497cb5a6)


perhaps speak to Diya if she can suggest a better design.

Reply:
@scmmishra Updated with the suggestion. When we hover the background, a remove button will appear.
<img width="617" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/30b13273-3631-4831-bfd8-ca62f9c555cc">

Reply:
Can we adjust the x spacing of that to make it even, should look like a square. Also, the icon, we probably need a background for it. It would be worth making that a button with `rounded-full bg-red-100` and the `x` svg in the center

Reply:
- In the empty state we can keep the "add emoji" icon.
- We should make the box size square.
- The cross can have a background and should be in a neutral color. I've used slate-10 for strokes, and slate-3 for background
![image](https://github.com/chatwoot/chatwoot/assets/69632863/759a843e-b796-4aa9-bcca-52a7b4dae7a8)
![image](https://github.com/chatwoot/chatwoot/assets/69632863/8146bfff-5c58-4814-9687-967a818f5be6)

Reply:
Empty state:
![image](https://github.com/chatwoot/chatwoot/assets/69632863/a19117ef-22a7-4fc2-a919-c91b9eb9e3e2)